doozer/doozerlib/cli/images_health.py

```diff
@@ -18,6 +20,28 @@
 now_unix_ts = int(round(time.time() * 1000))  # millis since the epoch
 
 
+def generate_art_dash_history_link(dg_name, build_record, runtime):
+    base_url = "https://art-dash.engineering.redhat.com/dashboard/build/history"
+
+    # Adjusting to tuple indexing for build_record
+    task_id, _, _, _ = build_record
+
+    # Extract the part after the last slash in dg_name
+    formatted_dg_name = dg_name.split("/")[-1]
+
+    params = {
+        "group": runtime.group_config.name,
+        "dg_name": formatted_dg_name,
+        "brew_task_id": task_id,
+    }
+
+    # Exclude any empty or irrelevant parameters
+    filtered_params = {k: v for k, v in params.items() if v}
```

Comment:
in which case item will be empty? if the param is empty we should return earlier and raise bad params, but when it would happened

Reply:
ohhh, right that makes sense. Will implement this.

Reply:
In normal operation, these parameters are expected to be present and correctly populated. However, the checks for empty parameters are a precautionary measure.